README.md

```diff
@@ -1,2 +1,68 @@
 # rosbot-xl-snap
-Building rosbot-xl snap for rosbot_xl_ros package
+
+This snap packages the [`rosbot_xl_ros`](https://github.com/husarion/rosbot_xl_ros) package.
+It thus conveniently offers all the ROS 2 stack necessary to bring up the [ROSbot-XL](https://husarion.com/manuals/rosbot-xl/) robot,
+including IMU driver, robot state publisher, joint state publisher, controllers and more.
+
+[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/rosbot-xl)
+
+## Installation
+
+Install the snap as follows,
+
+```bash
+snap install rosbot-xl
```

Comment:
CI is pushing on candidate and edge, and the uploaded snap is only available on `beta` should the channel be specified?

Reply:
Also sudo is required.